libsignal-service/src/gv2/models.rs

```diff
@@ -0,0 +1,30 @@
+use std::collections::HashMap;
+
+use serde::Deserialize;
+use zkgroup::auth::AuthCredentialResponse;
+
+#[derive(Debug, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TemporalCredential {
+    credential: String,
+    redemption_time: i64,
```

Comment:
Should we use `chrono::DateTime` to signify when it expires instead?

Reply:
This is what feeds the credential in the `HashMap<i64, ...>` and directly comes from the API, so I'm not sure there's any benefit to it?